python/tests/compat/test_histogram_events.py

```diff
@@ -0,0 +1,41 @@
+from comparison import Comparison
+from mantid_data_helper import mantid_is_available
+import pytest
+import mantid.simpleapi as sapi
+import scipp.compat.mantid as mantid
+import scipp as sc
+import numpy as np
+
+
+class HistogramEventsTest(Comparison):
+    def __init__(self):
+        super(HistogramEventsTest, self).__init__('histogram_events_test')
+
+    @property
+    def _filenames(self):
+        return {
+            "CNCS_51936_event.nxs": ("5ba401e489260a44374b5be12b780911", "MD5")
+        }
+
+    def _run_mantid(self, **kwargs):
+        # Note Mantid rebin inclusive of last bin boundary
+        out = sapi.Rebin(kwargs['workspace'],
+                         Params=[0, 10, 1000],
+                         PreserveEvents=False,
+                         StoreInADS=False)
+        return mantid.from_mantid(out)
+
+    def _run_scipp(self, **kwargs):
+        return sc.histogram(
+            kwargs['data_array'],
+            sc.Variable(dims=['tof'],
+                        values=np.arange(0, 1010, 10),
+                        dtype=sc.dtype.float64,
+                        unit=sc.units.us))
```

Comment:
This function signature is odd:
- Why do we need to use `kwards['data_array']`? It is unclear why the arg is named `data_array`. Can we just forward the `kwargs`, and ensure this can be set when using the class?
- Hard-coding the extra params in the class will make testing with multiple different params cumbersome.

I think as a user of the comparison class I would like to do something like this:

```python
@pytest.mark.parametrize("mantid_args, scipp_args", [(<first dict of mantid keyword args>, <first dict of scipp keyword args>), (<second dict of ...> , ...), ...])
def test_histogram_events():
  # not sure how to best define the name of the keyword arg initialized with the loaded file?
  rebin = HistogramEventsTest(filenames=['a.nxs', 'b.nxs'])
  rebin.check(mantid_args=mantid_args, scipp_args=scipp_args)
```